pkg/security/security.go

```diff
@@ -42,6 +42,17 @@ const (
 	// DefaultRootCertFilePath is the well-known path for an existing root certificate file
 	DefaultRootCertFilePath = "./etc/certs/root-cert.pem"
 
+	// GkeWorkloadCertChainFilePath is the well-known path for the GKE workload certificate chain file.
+	// Quoted from https://cloud.google.com/traffic-director/docs/security-proxyless-setup#create-service:
+	// "On creation, each Pod gets a volume at /var/run/secrets/workload-spiffe-credentials."
+	GkeWorkloadCertChainFilePath = "./var/run/secrets/workload-spiffe-credentials/certificates.pem"
```

Comment:
This path is actually not a special value that is relevant to Istio. That only impacts the gke WI webhook - something we would have no need for given we already have our own webhook and will configure the volume ourself.

I suggest we use a more standard path - there is no need for anything GKE specific at all as far as I can tell. I recall we even have some similar code to automatically read from /etc/certs if it's present. It's possible we may not even need any code changes if we do that - or if we do, like less (perhaps different filename if we cannot configure that)

Reply:
The code has been revised to read from ./etc/certs.  The cert and key file names of GKE workload certificates differ from those of the default file mounted cert:  the cert and key file names of GKE workload certificate are "certificates.pem", "private_key.pem", and "ca_certificates.pem", whereas those of the default file mounted cert are "cert-chain.pem", "key.pem", and "root-cert.pem".

Reply:
The path /var/run/secrets/workload-spiffe-credentials/{certifcates.pem,private_key.pem,ca_certificates.pem} is shaping up to be a cross-product standard (envoy and grpc directly support that path).  If possible, we should keep that path the same.

Reply:
The PR has been revised to use the standard path of GKE workload certificate (./var/run/secrets/workload-spiffe-credentials).

Reply:
Can you provide some reference to it becoming standard? I don't see any mention of it in the envoy or grpc repos. If it is a standard then sounds great, but we should link to a doc about it.

Reply:
@ahmedtd Do you have a reference link for it?